### ISSUE TITLE
[TSDoc Home Page]: Screen Reader is not announcing positions for links in navigation landmark in TSDoc home page.

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#20]
**Test Environment:**
OS Build: Windows 10
Version: 2004 (OS Build 21327.1000)
Browser: Edge Dev
Version 91.0.825.0 (Official build) dev (64-bit)
URL: https://tsdoc.org/
User: V-

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Turn on Narrator
4) Navigate to 'Intro' , 'Tags' , 'Github' , 'Help' and 'Playground' links in navigation landmark in TSDoc home page and observe the issue.

**Actual:**
Screen Reader is not announcing positions for  'Intro', 'Tags' , 'GitHub' , 'Help' and 'Playground' links in navigation landmark in TSDoc home page.

**Expected:**
Screen Reader should announce positions for  'Intro', 'Tags' , 'GitHub' , 'Help' and 'Playground' links in navigation landmark in TSDoc home page.

**User Experience:** 
Knowing how many items are in a list is extremely useful to assistive technology users as they can know if they should attempt to skip the list, navigate to the end, help them recall placement of certain items, or orient themselves within the list. 

**MAS Reference:**
MAS1.3.1-Info and Relationships

https://user-images.githubusercontent.com/85154417/120748688-d3532900-c520-11eb-8d31-64e0132d575a.mp4
